dbld/images/devshell.dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM balabit/syslog-ng-tarball:latest
+ARG DOCKER_IMAGE_PREFIX=balabit
```

Comment:
The default value is optional in case of the Docker ARG command.

It makes it possible, to build the dockerfile directly, without DBLD. I am not sure if we want to preserve that capability (I tried.). It was a restriction from DockerHub.

Reply:
I see, I am convinced, thanks! :)